packages/system/suc-upgrade/suc-upgrade.sh

```diff
@@ -8,18 +8,18 @@ echo "SUC_VERSION: $SUC_VERSION"
 if [ "$FORCE" != "true" ]; then
     if [ -f "/etc/kairos-release" ]; then
       # shellcheck disable=SC1091
-      UPDATE_VERSION=$(source /etc/kairos-release && echo "${KAIROS_VERSION}")
+      UPDATE_VERSION=$(source /etc/kairos-release && echo "KAIROS_VERSION=${KAIROS_VERSION};KAIROS_SOFTWARE_VERSION=${KAIROS_SOFTWARE_VERSION}")
```

Comment:
maybe we could just merge both keys so it behaves as before. You are saying that before it would be VERSION+K3SVERSION so maybe a simpler thing would be to do 

```
UPDATE_VERSION=$(source /etc/kairos-release && echo "${KAIROS_VERSION}${KAIROS_SOFTWARE_VERSION}")
```

thoughs?